contracts/TreasuryManager.sol

```diff
@@ -28,24 +25,25 @@ contract TreasuryManager is
     /// @notice precision used to limit the amount of NOTE price impact (1e8 = 100%)
     uint256 internal constant NOTE_PURCHASE_LIMIT_PRECISION = 1e8;
 
+    // following constants are relevant only for mainnet
+    IERC20 public constant NOTE = IERC20(0xCFEAead4947f0705A14ec42aC3D44129E1Ef3eD5);
```

Comment:
Can you add an onlyMainnet modifier and revert when the NOTE reinvestment methods that are only valid on mainnet are called on other networks?

approveBalancer
setNOTEPurchaseLimit
claimBAL
investWETHAndNOTE